### ISSUE TITLE
[SPIKE] Analyze low income threshold

### ISSUE THREAD

**Issue by BethMattern** [usds/justice40-tool#1332]
Exploratory work

**Comment by emma-nechamkin** [comment]
TL;DR is I don't see much gain from changing the income threshold, but I do generally wonder about how we are combining thresholds from each category (some of the components are not quite continuous, and some within category thresholds aren't tightly correlated, so we are left with, for example, a lot more tracts from some thresholds than from others). In re low income, one potential option is to have a "sliding" income threshold (that is, if you're in the 95th percentile for burden, for example, you could be in the 55th percentile for income). This has some benefit in terms of the census tracts we'd be adding, and could get rid of some of the discontinuity / step-wise nature of thresholding... but it's much harder to think about and explain.

**Comment by BethMattern** [comment]
@emma-nechamkin it sounds like we shouldn't recommend a change to the low income threshold. However, I'd like to learn more about this statement - "but I do generally wonder about how we are combining thresholds from each category (some of the components are not quite continuous, and some within category thresholds aren't tightly correlated, so we are left with, for example, a lot more tracts from some thresholds than from others)"

**Comment by emma-nechamkin** [comment]
@BethMattern so here, what I mean is there's a cliff -- so if you are in the 99th percentile for all categories of our environmental indicators, but 64th percentile for low income, you WONT be a DAC... but if you're in 90th percentile for just one and 65th percentile for low income, you WILL be a DAC. I think we can look into this a bit more when we analyze how many thresholds a tract exceeds. Wdyt?
